source/dubregistry/dbcontroller.d

```diff
@@ -132,12 +132,26 @@ class DbController {
 
 	bool hasVersion(string packname, string ver)
 	{
-		auto packbson = Bson(packname);
-		auto verbson = serializeToBson(["$elemMatch": ["version": ver]]);
-		auto ret = m_packages.findOne(["name": packbson, "versions" : verbson], ["_id": true]);
+		auto ret = m_packages.findOne(["name": packname, "versions.version" : ver], ["_id": true]);
```

Comment:
I'm trying to remember if there way any subtle reason for the `$elemMatch`, but don't really see any...